src/main/kotlin/com/sourcegraph/cody/agent/CodyAgent.kt

```diff
@@ -287,7 +287,7 @@ private constructor(
     }
 
     private fun traceWriter(): PrintWriter? {
-      val tracePath = System.getProperty("cody-agent.trace-path", "")
+      val tracePath = System.getProperty("cody.agent.trace-path", "")
```

Comment:
We should not change this as we have existing (internal dev) usage of this property (example: in my VM options).

Reply:
Ok switched it to kebab-case again